### PR TITLE
configuration for journals frontend app

### PR DESCRIPTION
**Pull request by bill-filler**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -327,6 +327,7 @@ ECOMMERCE_SOCIAL_AUTH_REDIRECT_IS_HTTPS: true
 ecommerce_create_demo_data: true
 
 JOURNALS_URL_ROOT: "https://journals-{{ EDXAPP_LMS_BASE }}"
+JOURNALS_FRONTEND_URL: "https://journalsapp-{{ EDXAPP_LMS_BASE }}"
```
Comment (tuchfarber) [inline util/jenkins/ansible-provision.sh:330]: I realize we can change it, but we might want to have a conversation with architecture about how we want to name these. I don't have any preferences on naming scheme currently, so I'm fine merging as is.

File: playbooks/roles/journals/defaults/main.yml [hunk 2]
```diff
@@ -57,6 +57,7 @@ JOURNALS_MEMCACHE: [ 'memcache' ]
 JOURNALS_VERSION: "master"
 JOURNALS_DJANGO_SETTINGS_MODULE: "journals.settings.production"
 JOURNALS_URL_ROOT: 'http://localhost:{{ JOURNALS_NGINX_PORT }}'
+JOURNALS_FRONTEND_URL: 'http://localhost:1991'
```
Comment (tuchfarber) [inline playbooks/roles/journals/defaults/main.yml:60]: Fine with merging as is. Don't know if we should pick a new port for the all the frontends as the cookie cutter defaults to 1991 and we won't be able to run multiple at once. That's more of a long term plan though (aka how we're going to run devstacks with 8+ separate frontends). Something we'll need to think about moving forward.

**Review by tuchfarber — APPROVED** [review]

File: util/jenkins/ansible-provision.sh (hunk 1 above)
Comment (bill-filler) [reply util/jenkins/ansible-provision.sh:330]: these actually get ovewritten by the whitelabel task which gets the proper names out of themes sandbox config

File: playbooks/roles/journals/defaults/main.yml (hunk 2 above)
Comment (bill-filler) [reply playbooks/roles/journals/defaults/main.yml:60]: same comment as above

**Review by fredsmith — APPROVED** [review]

**Pull request merged**